openMINDS_pipeline/resolver.py

```diff
@@ -71,12 +71,27 @@ def _schemas_by_category(schemas: List[SchemaStructure]) -> Dict[str, List[str]]
             for c in s.categories:
                 if c not in result:
                     result[c] = []
-                result[c].append(s.type)
+                # Only SANDS is in uppercase in schema_group
+                if s.schema_group == "SANDS":
+                    result[c].append('sands:' + s.type)
+                else:
+                    result[c].append(s.schema_group + ':' + s.type)
```

Comment:
Could this be changed to 
`result[c].append(s.schema_group.lower() + ':' + s.type)
`

Reply:
It can't be applied here, because of the prefixes specimenPrep and controlledTerms, but I can do a replace instead of a condition.

Reply:
Ok, then maybe it is clearer to leave as is